### PR TITLE
UI Fixes on metadata filtering and checkbox dropdown

### PR DESCRIPTION
**Pull request by sadelokiki**
# Description

This PR fixes some irregularities on metadata filtering and removes dropdown icon in check-all box 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix a feature that knowingly causes existing functionality to not work as expected)

# Test and Reproduce
Outline the steps to test or reproduce the PR here.

1. 

## Impacted Areas in Application
List general components of the application that this PR will affect:

Which pages?

- [ ] Login Screen
- [ ] Single Upload Page
- [ ] Batch Upload Page
- [ ] All Projects Page
- [ ] Sample Details Page
- [ ] Report Page

Specific components 

* 

# Testing Script:

###Login Page
* Successfully login to the application and get taken to the project page

###Single Upload
* Using an AWS path that doesn't exist throws an error
* Succesfully uploaded files from a folder (use test host)
* Uploading a duplicate file throws an error
* Pipeline gets initiated successfully 

###Batch Upload
* Using an AWS path that doesn't exist throws an error
* Succesfully uploaded files from a folder (use test host)
* Navigate to batch upload settings page after upload 
* Host selection from previous page still selected 
* Uploading a duplicate file throws a warning
* Pipeline gets initiated successfully 

###All Project Page
* Switch between projects 
* Search for a sample
* Navigate through different pages of samples
* Select Upload new sample button
* Select sample to see generated report

###Sample Details Page
* Information is consistent from Project Page
* Notes are editable and save successfully 
* Can navigate to report page 

###Report Page
* Data loads succesfully 
* Collapse and Uncollapse button works
* Clicking on the taxonomy name gives you total reads for that taxon
* Category filter works
* Genus Search works
* Filters work as expected
* Disabling Filters works
* Filtering by column works 

# Checklist:

- [ ] I have run through the testing script to make sure current functionality is unchanged
- [ ] I have done relevant tests that prove my fix is effective or that my feature works
- [ ] I have spent time testing out edge cases for my feature
- [ ] I have updated the test script or pull request template if necessary
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by yunfangjuan** [comment]
1. I still have trouble closing the metadata filter box. Had to refresh the page to close it. 
2. When I filter on Tissue type '-', I got nothing. It should show all the samples with tissue type == nil

File: app/assets/src/components/Samples.jsx [hunk 1]
```diff
@@ -952,7 +959,7 @@ class Samples extends React.Component {
       <div>
         <div className="col s2 metadata">
             <div className='metadata-dropdown' onClick={this.displayMetaDataDropdown}>
-            Metadata </div><i className="fa fa-angle-down" onClick={this.displayMetaDataDropdown}></i>
+            Filter </div><i className="fa fa-angle-down" onClick={this.displayMetaDataDropdown}></i>
```
Comment (yunfangjuan) [inline app/assets/src/components/Samples.jsx]: do you need both 961 and 962?

**Comment by sadelokiki** [comment]
@yunfangjuan, the tissue-type issue has been fixed in this PR: https://github.com/chanzuckerberg/idseq-web/pull/788.

**Comment by yunfangjuan** [comment]
1. For metadata filtering could you make it not fetching samples if there’s no change when people open/close the filter box?
2. This is a question for @eggerdesigns but we don't really show people what filters are turned on because they are hiding behind the box and that makes the filtered results very confusing. I suppose we are not merging this PR to master directly once it's approved?

**Comment by yunfangjuan** [comment]
Also, here is a bug: 
When you clicking on some filters, the url changes. But when you hit reload, it shows you different results. I think filters specified in the url needs to be respected.

File: db/schema.rb [hunk 2]
```diff
@@ -34,7 +34,7 @@
     t.index ["pipeline_output_id"], name: "index_backgrounds_pipeline_outputs_on_pipeline_output_id"
   end
 
-  create_table "backgrounds_pipeline_runs", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=latin1" do |t|
```
Comment (yunfangjuan) [inline db/schema.rb]: did you make any schema changes? If not, don't include this file.
